### PR TITLE
rpc: add (*Client).GetDesignatedByRole method

### LINKED ISSUE #1835: RPC client function for role management contract
`getDesignatedByRole` should be easily accessible via RPC (returning proper set of keys).

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #1835

### CODE REVIEW

File: pkg/rpc/client/native.go [hunk 1]
```diff
@@ -34,3 +39,54 @@ func (c *Client) GetGasPerBlock() (int64, error) {
 	}
 	return c.invokeNativeGetMethod(neoHash, "getGasPerBlock")
 }
+
+// GetDesignatedByRole invokes `getDesignatedByRole` method on a native RoleManagement contract.
+func (c *Client) GetDesignatedByRole(role native.Role, index uint64) (keys.PublicKeys, error) {
```
Comment (roman-khimov) [inline pkg/rpc/client/native.go:44]: `uint32`? It's a block number and they're 32-bit.

File: pkg/rpc/client/native.go [hunk 2]
```diff
@@ -3,9 +3,14 @@ package client
 // Various non-policy things from native contracts.
 
 import (
+	"crypto/elliptic"
 	"fmt"
 
+	"github.com/nspcc-dev/neo-go/pkg/core/native"
```
Comment (roman-khimov) [inline pkg/rpc/client/native.go:9]: Not a good dependency for `rpc/client`, although I see that it's already there for `native.NotaryVerificationPrice`.

Comment (AnnaShaleva) [reply pkg/rpc/client/native.go:9]: We may duplicate these constants inside the RPC client, but I'm not sure it's a good idea.

Comment (roman-khimov) [reply pkg/rpc/client/native.go:9]: Constants can always be moved into package of their own.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**